src/command.js

```diff
@@ -25,11 +26,12 @@ export default () => {
     if (!isAddress(address)) { throw new Error(`${address} is not valid address`); }
   });
 
-  if (program.from > 0 && !isValidBlockNumber(program.from)) { throw new Error(`${program.from} is not valid block number`); }
+  if (program.from !== defaultBlockNumber && !isValidBlockNumber(program.from)) { throw new Error(`${program.from} is not valid block number`); }
 
-  if (program.to > 0 && !isValidBlockNumber(program.to)) { throw new Error(`${program.to} is not valid block number`); }
+  if (program.to !== defaultBlockNumber && !isValidBlockNumber(program.to)) { throw new Error(`${program.to} is not valid block number`); }
 
-  if ((program.from > 0 && program.to > 0) && program.from > program.to) {
+  if ((program.from !== defaultBlockNumber && program.to !== defaultBlockNumber)
```

Comment:
I think this is redundant 
`(program.from !== defaultBlockNumber && program.to !== defaultBlockNumber)` 
as if both were default values `program.from` will never be bigger than `program.to`